src/vm/js/make-release.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs');
+const path = require('path');
+
+const releaseDir = 'release';
+
+const version = process.argv[2];
+
+if (!version) {
+  console.error('USAGE: node make-release.js VERSION');
+  process.exit();
+}
+
+try {
+  fs.mkdirSync(releaseDir);
+} catch (e) {
+  if (e.code !== 'EEXIST') throw e;
+}
+
+const precompiledPerl6 = '/home/pmurias/rakudo/node_modules';
```

Comment:
There are a couple instances of hard coded absolute paths in this file with your home path